assert/assertions.go

```diff
@@ -22,7 +22,7 @@ import (
 	yaml "gopkg.in/yaml.v2"
 )
 
-//go:generate go run ../_codegen/main.go -output-package=assert -template=assertion_format.go.tmpl
+//go:generate sh -c "cd ../_codegen && go build && cd - && ../_codegen/_codegen -output-package=assert -template=assertion_format.go.tmpl"
```

Comment:
These changes are required because `go run` can no longer be used with a relative path now that the `_codegen` is its own module. This builds the _codegen binary, then executes it from the current directory.